components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabConfigHelper.kt

```diff
@@ -38,6 +38,8 @@ import mozilla.components.support.utils.toSafeIntent
 import java.util.UUID
 import kotlin.math.max
 
+internal const val EXTRA_NAVIGATION_BAR_COLOR = "androidx.browser.customtabs.extra.NAVIGATION_BAR_COLOR"
```

Comment:
Shouldn't this go into the `browser-session`'s `CustomTabConfig` with the other intent extra keys?